src/main/resources/com/rultor/agents/daemons/end.sh

```diff
@@ -1,4 +1,9 @@
 #!/usr/bin/env bash
+#@todo #754:30min Remove the deprecation code from the shell.
+# Move the code to Java. May be
```

Comment:
@original-brownbear Now the next question :) Sorry for keep asking them. (May be the [link](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html) give the guts to ask them.

I can create a profile image like the test you have given, and equate that. But how do  I get the current profile?

How do i get the current Profile in `com.rultor.agents.daemons.EndsDaemon`?